desc/coils.py

```diff
@@ -1166,16 +1166,32 @@ def from_symmetry(cls, coils, NFP=1, sym=False):
         # check toroidal extent of coils to be repeated
         maxphi = 2 * np.pi / NFP / (sym + 1)
         data = coils.compute("phi")
+        check_coil_dists = False
         for i, cdata in enumerate(data):
-            errorif(
+            warnif(
                 np.any(cdata["phi"] > maxphi),
-                ValueError,
-                f"coil {i} exceeds the toroidal extent for NFP={NFP} and sym={sym}",
+                UserWarning,
+                f"coil {i} crosses the phi=2pi/{NFP*(sym+1)} symmetry plane"
+                + f" for NFP={NFP} and sym={sym},"
+                + "it is recommended to check the coil-coil separation with method"
```

Comment:
given that we check this automatically below i don't think we need these warnings here (maybe we just store the indices of offending coils, and include those in the warning below?)